pkg/operator/starter.go

```diff
@@ -160,7 +154,10 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	if err != nil {
 		return err
 	}
-	migrator := migrators.NewInProcessMigrator(dynamicClientForMigration, kubeClient.Discovery())
+
+	migrationClient := kubemigratorclient.NewForConfigOrDie(controllerContext.KubeConfig)
```

Comment:
based on the previous comment https://github.com/openshift/cluster-kube-apiserver-operator/pull/713/files#r366551501 we need to set QPS/Burst

Reply:
This is just for the CRD, not the actual migrated objects. We have to verify qps/burst in the migrator.

Reply:
ic